amt/site/templates/macros/render.html.j2

```diff
@@ -1,10 +1,16 @@
-{% macro column(status, translations, tasks_service) -%}
+{% macro column(project, status, translations, tasks_service) -%}
 <div class="col span_1_of_4">
     <h3 class="text-center-horizontal margin-bottom--small">{{ translations[status.name] }}</h3>
     <div class="progress_cards_container sortable" data-id="{{ status.id }}" id="column-{{ status.id }}">
+        {% if project and project.id %}
+        {% for task in tasks_service.get_tasks_for_project(project.id, status.id) %}
+        {{ render_task_card_full(task) }}
+        {% endfor %}
+        {% else %}
         {% for task in tasks_service.get_tasks(status.id) %}
```

Comment:
We may need to rethink the demo feature or update the current one so a 'demo project' is created. Eventually, a task should (must) never not have a project_id so this code would become obsolete.

Reply:
Agree for a 100%. But for now it seems ok.